docs/guides/using-environments.md

```diff
@@ -9,11 +9,19 @@ If you'd like to evaluate feature gates, dynamic configs, and/or experiments to
 
 ## SDK Usage
 
-When initializing the SDK, set the environment via the `StatsigOptions` parameter.  All SDKs accept an SDK key and an (optional) `StatsigOptions` dictionary of parameters.
+There are two main levers for setting environment- 
+1. **Environment-specific SDK keys**- to control which rules are downloaded to a given SDK 
+2. **Environment tier at SDK initialization**- to control which rules are evaluated  
```

Comment:
I would word this as more like it is used for evaluating rules not just deciding what rules are evaluated